### ISSUE TITLE
Unconditional printing to stderr

### ISSUE THREAD

**Issue by josalem** [Sysinternals/SysinternalsEBPF#1]
https://github.com/Sysinternals/SysinternalsEBPF/blob/b9035cf7780c9c733f4be06ff29f7b4fc447b395/telemetryLoader.c#L1426

The library unconditionally prints to `stderr`, which can be problematic for consumption. If you have a TUI for instance, this can corrupt the TUI output if the user doesn't redirect stderr.

You can work around this by using `dup2` to point stderr at `/dev/null`, but it would be nice if the library used some kind of logging library or put `fprintf(stderr...)` behind a config flag.

**Comment by kesheldr** [comment]
Agreed - will put this on the list. Feel free to suggest a change though if you happen to be fixing your local copy. :)
